katuma.yml

```diff
@@ -14,3 +14,14 @@
         log_path: "{{ shared_path }}/log"
         pids_path: "{{ shared_path }}/pids"
         ofn_route_override: "/admin/reports/order_cycle_management"
+
+    - role: jdauphant.nginx
```

Comment:
maybe import task https://github.com/jdauphant/ansible-role-nginx/blob/master/tasks/configuration.yml

Reply:
No luck. I created https://github.com/coopdevs/katuma-provisioning/issues/16 so that I could merge this.